Application/Platform/Gatekeeper/Authentication/Frontend.php

```diff
@@ -522,13 +626,16 @@ private function getIdentificationLayout($Content)
     }
 
     /**
-     * @param int $tblAccount
-     * @param int $tblIdentification
-     * @param int $doAccept 0|1
+     * @param int  $tblAccount
+     * @param int  $tblIdentification
+     * @param int  $doAccept 0|1
+     * @param bool $isCookieAvailable
+     *
      * @return Stage
      */
-    public function frontendIdentificationAgb($tblAccount, $tblIdentification, $doAccept = 0)
+    public function frontendIdentificationAgb($tblAccount, $tblIdentification, $doAccept = 0, $isCookieAvailable = false)
```

Comment:
Codacy found an issue: [The method frontendIdentificationAgb() has 137 lines of code. Current threshold is set to 100. Avoid really long methods.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)